crates/gen-c/src/lib.rs

```diff
@@ -1354,6 +1348,15 @@ impl Bindgen for FunctionBindgen<'_> {
                 }
             }
 
+            Instruction::UnitLower => {}
+            Instruction::UnitLift => {
+                // TODO:
+                // results.push("None".to_string());
```

Comment:
Should we `todo!` this?

Reply:
Never mind, I see that this will be implemented with a placeholder result in #206.

Reply:
Indeed yeah I wasn't sure how this was gonna pan out but it ended up getting solve well-ish in the next PR